Content.Shared/Construction/Conditions/WallmountCondition.cs

```diff
@@ -26,9 +26,12 @@ public bool Condition(EntityUid user, EntityCoordinates location, Direction dire
 
             // find direction from user to blueprint
             var userToObject = (objWorldPosition - userWorldPosition);
+            // get direction of the grid being placed on as an offset.
+            var gridRotation = IoCManager.Resolve<IEntityManager>().GetComponent<TransformComponent>(location.EntityId).WorldRotation;
```

Comment:
All of the IEntityManager resolves in this method can be replaced with the entManager that is already resolved above.